CMakeLists.txt

```diff
@@ -122,7 +122,22 @@ endif()
 if( NOT CMAKE_BUILD_TYPE )
   set( CMAKE_BUILD_TYPE Release )
 endif()
-set( FALLBACK_VERSION "Uncrustify-0.64_f" )
+
+# get a fallback version string from the package.json file to guard against
+# failure of the make_version.py script
+set(FALLBACK_VERSION "")
+file(READ "package.json" content)
+string(REPLACE "\n" ";" lines "${content}")
+foreach(line ${lines})
+    string(STRIP "${line}" line)
+    if("${line}" MATCHES "^\"version\" *: *\"([0-9.]+)\"")
```

Comment:
How can this work as the line starts with two spaces?
I would also replace whitespaces with \s character just to be sure (if supported).
And why not take any (.*) version string and enforce a specific format ?